brigade/static/scss/core/_variables.scss

```diff
@@ -7,16 +7,23 @@ $color-black: #000;
 $color-gray-dark: #5a5a5a;
 $color-white: #fff;
 
+$color-red: #cd1f42;
+$color-light-red: #d5405e;
+$color-dark-red: #bd1b3c;
+$color-blue: #399ed2;
+$color-light-blue: #57acd9;
+$color-dark-blue: #3591c0;
```

Comment:
Hm, I think we should just import the upstream styles:

```scss
@import '~@codeforamerica/style/_sass/core/_variables.scss';
```

and just override whatever is necessary?

(Note: there is currently one override in `style.scss` but I think it could move into the variables file. (And we'd have to import the variables file first so that the overridden values would be used subsequently.)

Reply:
Good point, that does seem like a better approach.